easybuild/easyconfigs/h/HBase/HBase-1.0.2.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PackedBinary'
+
+name = 'HBase'
+version = '1.0.2'
+
+homepage = 'http://hbase.apache.org/'
+description = """Apache HBase. is the Hadoop database, a distributed, scalable, big data store. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['%(namelower)s-%(version)s-bin.tar.gz']
+source_urls = ['http://www.us.apache.org/dist/hbase/hbase-1.0.2/']
+
+dependencies = [('Java', '1.7.0_80')]
+
+parallel = 1
+
+sanity_check_paths = {  
+    'files': ["bin/hbase"],  
+    'dirs': ["bin","conf","docs","hbase-webapps","lib"]  
```

Comment:
spaces after `,` would be nice (and including `bin` here isn't needed, since we already check that `bin/hbase` is there):

```python
sanity_check_paths = {  
    'files': ['bin/hbase'],
    'dirs': ['conf', 'docs', 'hbase-webapps', 'lib'],
}
```